lib/ruby_indexer/ruby_indexer.rb

```diff
@@ -4,6 +4,7 @@
 require "yaml"
 require "did_you_mean"
 
+require "ruby_indexer/lib/ruby_indexer/uri"
```

Comment:
As we near a separate release of the indexer, I expet we'll need to be careful not to patch Core classes a way which could affect other parts of someone codebase.

Reply:
If the Ruby LSP is being added as part of the composed bundle, there's no way it could impact someone's codebase because it's not a part of the main Gemfile.

And even if they do have it in the Gemfile, the default require for `gem "ruby-lsp"` doesn't do anything. You need to manually `require "ruby_lsp/internal"`, exactly for this reason (avoiding polluting the application's namespace).